drivers/clients_sub_pop/config/initializers/clients_sub_pop_feature.rb

```diff
@@ -0,0 +1,29 @@
+# The core app (or other drivers) can check the presence of the
+# ClientsSubPop driver with the following code snippet
+#
+#   do_something if RailsDrivers.loaded.include(:clients_sub_pop)
+#
+# use with caution!
+RailsDrivers.loaded << :clients_sub_pop
+
+AvailableSubPopulations.add_sub_population(
+  'All Clients',
+  :clients,
+  'ClientsSubPop::GrdaWarehouse::WarehouseReports::Dashboard::Clients',
+)
+
+GrdaWarehouse::Census.add_population(
+  population: :all_clients,
```

Comment:
Should this be `:clients`?

Reply:
Nope, at least not right now.  This is legacy, but it uses that to put the data in the right columns in the census tables.